### ISSUE TITLE
Picking up redgull doesnt work

### ISSUE THREAD

**Issue by silvercn** [ArmaLife/Altis#434]
If you buy redgull and die or get killed and try to pick it up, you wont be able to get it.

**Comment by Liam0** [comment]
The amount of time's ive seen people having problems picking up items o.O

is this a client/server or client/client sync issue? sometimes if i wait around it makes me able to pick it up but it's still anoying.

it feels like their not properly registered for some units in (some cases) i just find it hard to really figure out whats really going on especially during local testing.

Might be worth looking into an alternative way to use fn_dropitems or have the items spawn serverside directly?

**Comment by HyunsukBaek** [comment]
try window + T

**Comment by Liam0** [comment]
@HyunsukBaek What does windows + T do?

**Comment by silvercn** [comment]
Dunno, for me it does nothing. Anyway, i think best would be fetching items on ground and adding them to a list where you can select it.

**Comment by HyunsukBaek** [comment]
https://github.com/ArmaLife/Altis/issues/304

**Comment by Liam0** [comment]
@HyunsukBaek 

> is this a client/server or client/client sync issue? sometimes if i wait around it makes me able to pick it up but it's still anoying.

i don't actually think that ur shift or windows +t is the solution =)

**Comment by sargken** [comment]
The Issue is its using cursorTarget and unless you have it at the exact spot that that item is for it to detect the object it wont appear. I was asking it to be switched to nearestObject.

**Comment by Liam0** [comment]
@sargken, Using a loop&hint i have no problem at all with cursortarget on items sometimes by just pointing near it or at the furthest edges probably because of boundingbox even on the small cans.

**Comment by danielstuart14** [comment]
we don't use cursorTarget anymore, for items, we're using cursorObject, which has a faster response and doesn't have this problem of having to put the item in the exact spot.

I'll check it

**Comment by Liam0** [comment]
Yeah ur right @danielstuart14 but but that would hopefully just be even better at targeting objects which is why i dont really think the targeting is the problem x) but its odd and its too random -.-

**Comment by danielstuart14** [comment]
yeah, we can add a menu, that shows the items on ground and in player's inv.

**Comment by trinitynet** [comment]
or just have it pickup the nearest object within range say 3m or something like that?
